cookbooks/aws-parallelcluster-platform/test/controls/intel_hpc_spec.rb

```diff
@@ -137,3 +137,31 @@
     its('mode') { should cmp '0755' }
   end
 end
+
+control 'tag:intel_one_api_toolkits_configured' do
```

Comment:
In recipe `intel_hpc.rb` we have different types of installations. Does this test cover all of them?